src/fio/fioEngine.ts

```diff
@@ -1440,7 +1443,7 @@ export class FioEngine extends CurrencyEngine<FioPlugin> {
       }
     }
 
-    const { name, params }: { name: string, params: any } = otherParams.action
+    const { name, params }: { name: string; params: any } = otherParams.action
```

Comment:
Replace ; with ,

Reply:
Commas are actually incorrect TS syntax. TS uses newlines to separate type definitions for objects. But if each param is on the same line, they are separated by ;